packages/core/tests/oob.test.ts

```diff
@@ -715,6 +715,36 @@ describe('out of band', () => {
         new CredoError('There is no message in requests~attach supported by agent.')
       )
     })
+
+    test.only(`make two connections with ${HandshakeProtocol.DidExchange} by reusing the did from the first connection as the 'invitationDid' in oob invitation for the second connection`, async () => {
+      const outOfBandRecord1 = await faberAgent.oob.createInvitation({})
```

Comment:
I think it will work if you add something like this:
```
      const didRouting = await faberAgent.mediationRecipient.getRouting({})

      const { id: invitationDid } = await createPeerDidFromServices(
        faberAgent.context,
        [
          {
            id: 'didcomm',
            recipientKeys: [didRouting.recipientKey],
            routingKeys: didRouting.routingKeys,
            serviceEndpoint: didRouting.endpoints[0],
          },
        ],
        PeerDidNumAlgo.MultipleInceptionKeyWithoutDoc
      )

const outOfBandRecord1 = await faberAgent.oob.createInvitation({ invitationDid })
```

It's a bit verbose and feels somewhat 'hacky' but not that bad I think. But of course if there is a direct way to achieve a peer did creation that also creates its own routing it would make things easier for Credo users.

Reply:
What if we add a helper method to the OOB API?

```
/**
* Create a peer did that is capable for usage in DIDComm messaging. You can pass the created did to the
* `createInvitation` method as `invitationDid`.
*/
await agent.oob.createDidCommPeerDid({
  

})
```

That would be useful I think